InventoryBuilder.py

```diff
@@ -283,11 +288,15 @@ def create_nsxt_objects(self, vrops, target: str, token: str):
             nsxt_mgmt_plane.add_adapter(adapter)
 
         nsxt_mgmt_cluster, self.response_codes[target]["nsxt_mgmt_cluster"] = \
-            Vrops.get_nsxt_mgmt_cluster(vrops, target,token,[a.uuid for a in nsxt_adapter])
+            Vrops.get_nsxt_mgmt_cluster(vrops, target, token, [a.uuid for a in nsxt_adapter])
         nsxt_mgmt_nodes, self.response_codes[target]["nsxt_mgmt_nodes"] = \
             Vrops.get_nsxt_mgmt_nodes(vrops, target, token, [c.uuid for c in nsxt_mgmt_cluster])
         nsxt_mgmt_service, self.response_codes[target]["nsxt_mgmt_services"] = \
             Vrops.get_nsxt_mgmt_service(vrops, target, token, [n.uuid for n in nsxt_mgmt_nodes])
+        nsxt_transport_zones, self.response_codes[target]["nsxt_transport_zones"] = \
+            Vrops.get_nsxt_transport_zone(vrops, target, token, [c.uuid for c in nsxt_mgmt_cluster])
+        nsxt_transport_nodes, self.response_codes[target]["nsxt_transport_nodes"] = \
+            Vrops.get_nsxt_transport_node(vrops, target, token, [t.uuid for t in nsxt_transport_zones])
```

Comment:
```suggestion
            Vrops.get_nsxt_transport_node(vrops, target, token, [z.uuid for z in nsxt_transport_zones])
```
I would feel `z` more natural here, looking at the previous line pattern.